connector_prestashop/models/prestashop_backend/common.py

```diff
@@ -138,16 +137,86 @@ def _select_state(self):
     verbose = fields.Boolean(help="Output requests details in the logs")
     debug = fields.Boolean(help="Activate PrestaShop's webservice debug mode")
 
+
+    matching_product_template = fields.Boolean(string="Match product template")
+    
+    matching_product_ch = fields.Selection([('reference','Reference'),('ean13','Barcode')],string="Matching Field for product")
+    
+    matching_customer = fields.Boolean(string="Matching Customer", 
+                    help="The selected fields will be matched to the ref field \
+                        of the partner. Please adapt your datas consequently.")
+    matching_customer_ch = fields.Many2one(comodel_name='prestashop.partner.field'
```

Comment:
Yes for sure, you're right, I forgot to add the view part.
I've implemented this options there https://github.com/flotho/connector-prestashop/commits/9.0-matching-partners and there :  https://github.com/flotho/connector-prestashop/commits/9.0-matching-products
The purpose is to be able to bind already existing products / partner with the one coming from PS. I had this problem after a migration, aprtner where already there but the customer changed his PS installation and we redo all the stuf so we used matching to prevent duplicates. 
But you're right everything is not pushed in this PR but it will be in a close future.